src/Admin/AdminInterface.php

```diff
@@ -484,7 +484,7 @@ public function getTranslationDomain();
     /**
      * Return the form groups.
      *
-     * @return array
+     * @return array|false
```

Comment:
We could explain which cases makes this method to return `false`.

Reply:
Fixed